recipe/meta.yaml

```diff
@@ -1,79 +1,106 @@
-{% set name = "anaconda-cloud-auth" %}
-{% set version = "0.7.2" %}
+{% set version = "0.8.2" %}
 
 package:
-  name: {{ name|lower }}
+  name: anaconda-cloud-auth-split
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/anaconda_cloud_auth-{{ version }}.tar.gz
```

Comment:
We can't use the project on PyPI anymore? https://pypi.org/project/anaconda-auth/#history

Reply:
We can, but I was trying to refrain from using multiple sources. This [repo](https://github.com/anaconda/anaconda-auth) has both anaconda-auth and anaconda-cloud-auth.